DataWorkerService/Services/Implementations/OsuApiDataParserService.cs

```diff
@@ -81,7 +80,7 @@ public async Task ParseMatchAsync(Match match, MultiplayerMatch apiMatch)
                 {
                     // Scale up EZ scores
                     Score = mpScore.Mods.HasFlag(Mods.Easy)
-                        ? (int)(mpScore.Score * 1.75)
+                        ? (int)(mpScore.Score * 1.75) // TODO: parametrize
```

Comment:
```suggestion
                        ? (int)(mpScore.Score * 1.75)
```

Can convert to a constant instead at the top of the file.

Reply:
sure